### PR TITLE
Draft: Issue #3

### PR DESCRIPTION
**Pull request by Barrus00**
Pre-alpha paddle URDF model.
Pre-alpha paddle steering.

### CODE REVIEW

File: paddle/only_paddle.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+from paddle.abc_paddle import ABCPaddle
+
+
+class OnlyPaddle(ABCPaddle):
+    urdf_model = 'paddle/paddle.urdf'
+
+    # The following are the paddle important joints ids.
+    # These are hard coded values, so always make sure to check these after changing paddle urdf model.
+    move_axis_joints = {'x': 2, 'y': 1, 'z': 0}
+    rotate_axis_joints = {'x': 5, 'y': 4, 'z': 3}
```
Comment (matiutd888) [inline paddle/only_paddle.py:11]: Maybe also use full UPPER_CASE for those?

File: main.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
```
[the remark this answers is not in the document]
Comment (Barrus00) [reply main.py:1]: ok

File: main_on_classes.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+import pybullet as p
+import pybullet_data
+import time
+
+# start the simulation with a GUI (p.DIRECT is without GUI)
+p.connect(p.GUI)
+
+# Create a ball.
+ball_radius = 0.025
```
[the remark this answers is not in the document]
Comment (Barrus00) [reply main_on_classes.py:12]: ok, zgoda

File: main_on_classes.py [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    # Rotate the paddle around the x-axis
+    if x_steering != 0:
+        paddle.rotate_around_axis('x', x_steering * rotation_speed)
+
+    # Rotate the paddle around the y-axis
+    if y_steering != 0:
+        paddle.rotate_around_axis('y', y_steering * rotation_speed)
+
+    # step Simulation
+    p.stepSimulation()
```
[the remark this answers is not in the document]
Comment (Barrus00) [reply main_on_classes.py:101]: ok, zgoda

File: paddle/abc_paddle.py [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+    @abstractmethod
+    def move_by_vector(self, v):
+        pass
+
+    @abstractmethod
+    def move_to_position(self, p):
+        pass
+
+    #TODO: Nie wiem czy to w ogóle będzie potrzebne?
```
[the remark this answers is not in the document]
Comment (Barrus00) [reply paddle/abc_paddle.py:27]: ok, sorry

File: paddle/only_paddle.py (hunk 1 above)
Comment (Barrus00) [reply paddle/only_paddle.py:11]: ay, ay

**Pull request merged**